### DISCUSSION TITLE
How to set the composer version

### DISCUSSION THREAD

**Discussion by KptMonkey** [deployphp/deployer#2286] [Help needed]
Hi,

I am curious about how to set the composer version since my application requires v1.x. I added the line `set('composer_version', '1.10.17');` to the deploy.php file but during the process, it still curls the current version (2.0.7). 
It looks like that it gets overwritten in vendor.php. Since it's not a bug ticket, maybe there is another solution that will do the trick. I am not the only who has this problem https://stackoverflow.com/questions/64767217/php-deployer-change-composer-version (I added one comment to this thread)
I tried it with the deployer version 6.8 and the current beta

**Comment by antonmedv** [comment]
Just install the correct Composer version of the host and Deployer will use it.
In Deployer v7 it's possible to specify `composer_version` config and Deployer will download it and place under `.dep/composer.phar`.
